### DISCUSSION TITLE
[Feature Request] Change Download Folder Based on First Model Tag in Civitai

### DISCUSSION THREAD

**Discussion by serick4126** [BlafKing/sd-civitai-browser-plus#298] [Ideas]
When I download models from Civitai, I save them into subfolders based on their first tag (e.g., concept, character, style). 
Specifying the subfolder each time is cumbersome, so could you add a feature to set the default download path based on the first tag?

Sample Configuration:
{
    "concept": "\concept",
    "character": "\character",
    "style": "\style",
}

LoRA folder: c:\lora

Example:
If the first tag of a LoRA model is "style," the default download path would be c:\lora\style. 
If the first tag is "pose" (not defined in the configuration), the path would be c:\lora.

Thank you!

**Comment by Lumerica** [comment]
Maybe as long as this extension is able to index the category/concept, we can just change the behavior of the download folder structure via setings? So we just need it to index and we can either change via settings or even in the code of civitai_api.py and civitai_gui.py

Eg for me i wanted my folder to be specific like this so for me i could work with free text instead of pulling the concept name, if this extension is able to retrieve the category I would also have done that in the code xD

For now I have modified this in my file but same idea would work for you if we can indeed so something with category.
![image](https://github.com/user-attachments/assets/76e1f010-3fcb-4366-b5cd-16871c5e5748)
